### ISSUE TITLE
Not show the post of the user I muted, and not show the post of the user I follow, replying an user that I muted.

### ISSUE THREAD

**Issue by bitcoinuser** [vitorpamplona/amethyst#1175]
Hi,

When I mute someone, I don't want to see (Show Anyway) post and I don't want to see the message of the someone I follow replying an user that I muted. 
Conclusion: Not show the post of the user I muted, and not show the post of the user I follow, replying an user that I muted.

Thanks.

**Comment by davotoula** [comment]
What do we think here @vitorpamplona, @KotlinGeekDev?

I think it's technically possible but not sure it improves the user experience. Users could be missing out on replies from users they didn't mute which could be confusing.

Apropo feature requests, it would be helpful to have labels that can categorise feature requests and bugs. Something like `approved` (ready to be implemented), `won't fix`(for things that are as designed and/or will not improve amethyst), `clarification needed `(things that need more detail before they are approved), `beginner friendly` which are suitable for new contributors, `critical` for serious issues like app crashes or security issues, and `implemented` for things that are done and just waiting for a release?

**Comment by vitorpamplona** [comment]
Yeah, this is tricky because virtually all the other apps do continue displaying the thread, especially if the reply is made by a follow. 

I added you as a collaborator in the repo. I think you can edit labels now.

**Comment by davotoula** [comment]
@bitcoinuser have you considered the case that replies and comments in that thread from users that are not blocked may be hidden? 

Even if they are replying to you directly?

Do you have examples or nostr or other social media clients doing it this way?

**Comment by KotlinGeekDev** [comment]
> What do we think here @vitorpamplona, @KotlinGeekDev?
> 
> I think it's technically possible but not sure it improves the user experience. Users could be missing out on replies from users they didn't mute which could be confusing.
> 

Wow, not sure how I missed this. Yeah , you're right. People have already come to expect a certain UX with regards to replies, and this type of change may not be a good idea(unless we use a setting for it, maybe)

**Comment by vitorpamplona** [comment]
Closing as stale — this has been waiting on more info since 2024 and the mute/hidden-content handling has changed substantially since then (mute list, hidden-words filters, "show anyway" gating). If the behavior still bothers you on a current release, please open a fresh issue describing what you see now.

---
_Generated by [Claude Code](https://claude.ai/code)_
